node/db/src/rocks.rs

```diff
@@ -62,6 +62,7 @@ impl RocksDb {
         if !config.optimize_for_point_lookup.is_negative() {
             let cache_size = config.optimize_for_point_lookup as usize;
             let mut opts = BlockBasedOptions::default();
+            opts.set_block_size(config.block_size);
```

Comment:
If we set the `block_size` here this setting will be only picked when `optimize_for_point_lookup` is set (`!= -1`).

It would be more accurate to set it in the else branch too, in case the user disables `optimize_for_point_lookup` but still wants 2K blocks (otherwise it will be 4K block, the default value).